qemu/tests/cfg/multi_disk.cfg

```diff
@@ -56,17 +58,18 @@
             usbs += " default-ehci"
             usb_type_default-ehci = usb-ehci
         - virtio_scsi_variants:
+            only virtio_scsi
             # virtio-scsi driver support from RHEL6.3
             no RHEL.3 RHEL.4 RHEL.5 RHEL.6.0 RHEL.6.1 RHEL.6.2
             # Decrease length of the command
-            stg_image_size = 1M
+            stg_image_size = 100M
```

Comment:
Why increase  to 100M, I guest for some host this case will failed because host no enough space.  And I my mind, some sg device make by scsi_debug module in host , so if we increase it to 100M may cause host leak memory to continue our testing.
Just a reminder.

Thanks,
@xutian